.github/workflows/freebsd-12.yml

```diff
@@ -34,44 +34,22 @@ concurrency:
     format('{0}-{1}', github.workflow, github.ref) }}
   cancel-in-progress: true
 
-env:
-  CI_MAKE: make -f .gitlab.mk
-
 jobs:
-  freebsd:
+  freebsd-12:
     # Run on pull request only if the 'full-ci' label is set.
     if: github.event_name != 'pull_request' ||
         contains(github.event.pull_request.labels.*.name, 'full-ci')
 
-    runs-on: freebsd-sh
+    runs-on: freebsd-12-mcs
 
     strategy:
       fail-fast: false
 
     steps:
-      - name: set PATH to GIT of the newer version 2.9.0
-        run: echo "/usr/local/git/bin" | tee -a $GITHUB_PATH
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v2
+        with:
+          ref: ''
+          fetch-depth: 0
```

Comment:
Why it is not the same as in, say, `centos-7.yml`?

* `ref` should defaults to current branch and can be omitted.
* No `submodules: recursive`, hm.

Reply:
Ok. Did the same way.